### PR TITLE
bump ggcr & k8schain module versions

### PR DESCRIPTION
**Pull request by dprotaso**
I manually bumped the version of `github.com/google/go-containerregistry` & it's `k8schain` module.

`k8schain` was bumped to use the latest version (v1.19.7) of `github.com/vdemeester/k8s-pkg-credentialprovider`.

This should drop any transient k8s.io v1.18 deps - see https://github.com/tektoncd/triggers/pull/934#discussion_r566391743

```release-note
NONE
```

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -4,39 +4,52 @@ go 1.13
...
+	github.com/Azure/azure-sdk-for-go v50.2.0+incompatible // indirect
+	github.com/Azure/go-autorest/autorest v0.11.17 // indirect
+	github.com/Azure/go-autorest/autorest/adal v0.9.10 // indirect
+	github.com/Azure/go-autorest/autorest/to v0.4.0 // indirect
+	github.com/Azure/go-autorest/autorest/validation v0.3.1 // indirect
 	github.com/GoogleCloudPlatform/cloud-builders/gcs-fetcher v0.0.0-20191203181535-308b93ad1f39
+	github.com/aws/aws-sdk-go v1.37.1 // indirect
 	github.com/cloudevents/sdk-go/v2 v2.1.0
+	github.com/docker/cli v20.10.2+incompatible // indirect
+	github.com/docker/docker v20.10.2+incompatible // indirect
```
Comment (vdemeester) [inline go.mod:16]: That's a lot of new deps 😅

Comment (dprotaso) [reply go.mod:16]: the deps have been there before - unclear when the go toolchain adds it to the `go.mod` file with `//indirect`

**Pull request merged**